etc/config/cuda.amazon.properties

```diff
@@ -309,6 +309,12 @@ compiler.hipclang-rocm-50302.name=clang rocm-5.3.2
 compiler.hipclang-rocm-50302.nvdisasm=/opt/compiler-explorer/clang-rocm-50302/bin/llvm-objdump
 compiler.hipclang-rocm-50302.objdumper=/opt/compiler-explorer/clang-rocm-50302/bin/llvm-objdump
 compiler.hipclang-rocm-50302.options=-x hip --cuda-device-only -mllvm -amdgpu-early-inline-all=true -mllvm -amdgpu-function-calls=false -O3 -D__HIP_PLATFORM_AMD__=1 -D__HIP_PLATFORM_HCC__=1 -isystem /opt/compiler-explorer/libs/rocm/5.3.2/include --rocm-path=/opt/compiler-explorer/libs/rocm/5.3.2 --rocm-device-lib-path=/opt/compiler-explorer/clang-rocm-5.3.2/amdgcn/bitcode -include hip/hip_runtime.h
+compiler.hipclang-rocm-50700.exe=/opt/compiler-explorer/clang-rocm-5.7.0/bin/clang++
+compiler.hipclang-rocm-50700.semver=5.7.0
+compiler.hipclang-rocm-50700.name=clang rocm-5.7.0
+compiler.hipclang-rocm-50700.nvdisasm=/opt/compiler-explorer/clang-rocm-50700/bin/llvm-objdump
```

Comment:
These paths do not exist after installing clang rocm-5.7.0 -- I'm a little confused in honesty what this is and where it would come from. I see the other hipclangs etc have this kind of "clang in one location, llvm in another" - do you know where the latter might come from?

Reply:
It actually appears that _none_ of those llvm-objdumps exist!

```
Oct/10 00:59 admin-node~ 0.5s $ ls -l /opt/compiler-explorer/ | grep rocm
drwxr-xr-x   8 root     root      6144 Sep 21  2022 clang-rocm-4.5.2
drwxr-xr-x   8 root     root      6144 Sep 21  2022 clang-rocm-5.0.2
drwxr-xr-x   8 root     root      6144 Sep 21  2022 clang-rocm-5.1.3
drwxr-xr-x   8 root     root      6144 Sep 21  2022 clang-rocm-5.2.3
drwxr-xr-x   8 root     root      6144 Nov 17  2022 clang-rocm-5.3.2
drwxr-xr-x   8 root     root      6144 Oct  9 23:50 clang-rocm-5.7.0
lrwxrwxrwx   1 root     root        25 Oct  9 07:08 clang-rocm-trunk -> clang-rocm-trunk-20231009
drwxr-xr-x   8 root     root      6144 Oct  3 07:07 clang-rocm-trunk-20231003
drwxr-xr-x   8 root     root      6144 Oct  4 07:12 clang-rocm-trunk-20231004
drwxr-xr-x   8 root     root      6144 Oct  5 07:09 clang-rocm-trunk-20231005
drwxr-xr-x   8 root     root      6144 Oct  6 07:10 clang-rocm-trunk-20231006
drwxr-xr-x   8 root     root      6144 Oct  7 07:09 clang-rocm-trunk-20231007
drwxr-xr-x   8 root     root      6144 Oct  8 07:07 clang-rocm-trunk-20231008
drwxr-xr-x   8 root     root      6144 Oct  9 07:08 clang-rocm-trunk-20231009
```

I can only assume the `llvm-objdump` is unused. Anyone know?

Reply:
(I don't see a reason to block this if the others are equally as broken though....so will merge)

Reply:
Will definitely have to come back to this one if the others are broken too...

Reply:
I think in general the `nvdisasm` properties are probably out of place in the CUDA properties file.
And it might be best to just disable binary support for now. (so `supportsBinary=false` and `supportsBinaryObject=false`)

I also saw "--no-gpu-bundle-output" in https://github.com/compiler-explorer/compiler-explorer/blob/main/lib/compilers/clang.ts#L300 - would that be correct in case of CUDA? How do you export device code in this case?

Might be good to make a new issue to talk about how to support all of this.